src/WorkItemMigrator/WorkItemImport/WitClient/WitClientUtils.cs

```diff
@@ -651,6 +681,26 @@ private void RemoveSingleAttachmentFromWorkItemAndSave(WiAttachment att, WorkIte
                 }
             };
 
+            if (changedDate != default)
+            {
+                attachmentPatchDocument.Add(new JsonPatchOperation()
+                {
+                    Operation = Operation.Add,
+                    Path = "/fields/" + WiFieldReference.ChangedDate,
+                    Value = changedDate
+                });
+            }
+
+            if (changedBy != default)
+            {
+                attachmentPatchDocument.Add(new JsonPatchOperation()
+                {
+                    Operation = Operation.Add,
+                    Path = "/fields/" + WiFieldReference.ChangedBy,
+                    Value = changedBy
+                });
+            }
+
```

Comment:
This code is duplicated in AddSingleAttachmentToWorkItemAndSave(). Maybe you can extract this code block to a helper function?

Reply:
A private function in `WitClientUtils` should be fine for now :)

Reply:
I addressed this on https://github.com/solidify/jira-azuredevops-migrator/pull/648 by extracting the JsonPatchOperation creation to a separate utility class.